src/main/resources/application.yml

```diff
@@ -31,9 +34,9 @@ spring:
 #              packages: com.cs544.project.kafka.message
 logging:
   level:
-    root: ERROR
+    root: info
```

Comment:
should not commit this change to github, keep log level to ERROR since main will auto-deploy to production and we want production to only be ERROR level

could also change this into an env variable so we can set it automatically